src/diagram/models/variable.test.ts

```diff
@@ -697,6 +697,23 @@ describe("Variable", () => {
     expect(variable.computedUnitIncludingMessageAndError).toEqual({unit: "m", error: "incompatible units"});
   });
 
+  it("shows error when a cycle exists between two variables", () => {
+    const container = GenericContainer.create({
+      items: [
+        {id: "inputA", name: "a", inputs: ["inputA"], expression: "b"},
+        {id: "inputB", name: "b", inputs: ["inputB"], expression: "a"},
+      ]
+    });
+    const variable = container.items[1] as VariableType;
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const warn = jest.spyOn(console, "warn").mockImplementation(() => {});
```

Comment:
```suggestion
    const warn = jest.spyOn(console, "warn").mockImplementation(() => null);
```
In a situation like this I generally just use `null` which shouldn't require the `eslint-disable`.